### ISSUE TITLE
Build failure: python3Packages.ics

### ISSUE THREAD

**Issue by dbaynard** [NixOS/nixpkgs#305529]
### Steps To Reproduce

1. nix build -L 'github:NixOS/nixpkgs/cfd6b5fc90b15709b780a5a1619695a88505a176#python3Packages.ics'

### Build log

<details>

<summary>Preamble</summary>

```console
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/qp65vs1l5ckcs90zwm47pra6likn21n8-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/utils.py
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
substituteStream(): WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'requirements.txt')
substituteStream(): WARNING: pattern arrow\>=0.11\,\<0.15 doesn't match anything in file 'requirements.txt'
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
* Getting build dependencies for wheel...
/nix/store/4718wmk03wr3554kmf09vy80vkdjvq56-python3.11-setuptools-69.1.1/lib/python3.11/site-packages/setuptools/dist.py:472: SetuptoolsDeprecationWarning: Invalid dash-separated options
!!

        ********************************************************************************
        Usage of dash-separated 'license-file' will not be supported in future
        versions. Please use the underscore name 'license_file' instead.

        By 2024-Sep-26, you need to update your project and remove deprecated calls
        or your builds will no longer be supported.

        See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

!!
  opt = self.warn_dash_deprecation(opt, section)
running egg_info
creating ics.egg-info
writing ics.egg-info/PKG-INFO
writing dependency_links to ics.egg-info/dependency_links.txt
writing requirements to ics.egg-info/requires.txt
writing top-level names to ics.egg-info/top_level.txt
writing manifest file 'ics.egg-info/SOURCES.txt'
reading manifest file 'ics.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'meta.py'
adding license file 'LICENSE.rst'
writing manifest file 'ics.egg-info/SOURCES.txt'
* Building wheel...
/nix/store/4718wmk03wr3554kmf09vy80vkdjvq56-python3.11-setuptools-69.1.1/lib/python3.11/site-packages/setuptools/dist.py:472: SetuptoolsDeprecationWarning: Invalid dash-separated options
!!

        ********************************************************************************
        Usage of dash-separated 'license-file' will not be supported in future
        versions. Please use the underscore name 'license_file' instead.

        By 2024-Sep-26, you need to update your project and remove deprecated calls
        or your builds will no longer be supported.

        See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

!!
  opt = self.warn_dash_deprecation(opt, section)
running bdist_wheel
The [wheel] section is deprecated. Use [bdist_wheel] instead.
running build
running build_py
creating build
creating build/lib
creating build/lib/ics
copying ics/__init__.py -> build/lib/ics
copying ics/__meta__.py -> build/lib/ics
copying ics/attendee.py -> build/lib/ics
copying ics/component.py -> build/lib/ics
copying ics/event.py -> build/lib/ics
copying ics/icalendar.py -> build/lib/ics
copying ics/timeline.py -> build/lib/ics
copying ics/todo.py -> build/lib/ics
copying ics/tools.py -> build/lib/ics
copying ics/types.py -> build/lib/ics
copying ics/utils.py -> build/lib/ics
running egg_info
writing ics.egg-info/PKG-INFO
writing dependency_links to ics.egg-info/dependency_links.txt
writing requirements to ics.egg-info/requires.txt
writing top-level names to ics.egg-info/top_level.txt
reading manifest file 'ics.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'meta.py'
adding license file 'LICENSE.rst'
writing manifest file 'ics.egg-info/SOURCES.txt'
/nix/store/4718wmk03wr3554kmf09vy80vkdjvq56-python3.11-setuptools-69.1.1/lib/python3.11/site-packages/setuptools/command/build_py.py:207: _Warning: Package 'ics.alarm' is absent from the `packages` configuration.
!!

        ********************************************************************************
        ############################
        # Package would be ignored #
        ############################
        Python recognizes 'ics.alarm' as an importable package[^1],
        but it is absent from setuptools' `packages` configuration.

        This leads to an ambiguous overall configuration. If you want to distribute this
        package, please make sure that 'ics.alarm' is explicitly added
        to the `packages` configuration field.

        Alternatively, you can also rely on setuptools' discovery methods
        (for example by using `find_namespace_packages(...)`/`find_namespace:`
        instead of `find_packages(...)`/`find:`).

        You can read more about "package discovery" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

        If you don't want 'ics.alarm' to be distributed and are
        already explicitly excluding 'ics.alarm' via
        `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
        you can try to use `exclude_package_data`, or `include-package-data=False` in
        combination with a more fine grained `package-data` configuration.

        You can read more about "package data files" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/datafiles.html


        [^1]: For Python, any directory (with suitable naming) can be imported,
              even if it does not contain any `.py` files.
              On the other hand, currently there is no concept of package data
              directory, all directories are treated like packages.
        ********************************************************************************

!!
  check.warn(importable)
/nix/store/4718wmk03wr3554kmf09vy80vkdjvq56-python3.11-setuptools-69.1.1/lib/python3.11/site-packages/setuptools/command/build_py.py:207: _Warning: Package 'ics.grammar' is absent from the `packages` configuration.
!!

        ********************************************************************************
        ############################
        # Package would be ignored #
        ############################
        Python recognizes 'ics.grammar' as an importable package[^1],
        but it is absent from setuptools' `packages` configuration.

        This leads to an ambiguous overall configuration. If you want to distribute this
        package, please make sure that 'ics.grammar' is explicitly added
        to the `packages` configuration field.

        Alternatively, you can also rely on setuptools' discovery methods
        (for example by using `find_namespace_packages(...)`/`find_namespace:`
        instead of `find_packages(...)`/`find:`).

        You can read more about "package discovery" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

        If you don't want 'ics.grammar' to be distributed and are
        already explicitly excluding 'ics.grammar' via
        `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
        you can try to use `exclude_package_data`, or `include-package-data=False` in
        combination with a more fine grained `package-data` configuration.

        You can read more about "package data files" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/datafiles.html


        [^1]: For Python, any directory (with suitable naming) can be imported,
              even if it does not contain any `.py` files.
              On the other hand, currently there is no concept of package data
              directory, all directories are treated like packages.
        ********************************************************************************

!!
  check.warn(importable)
/nix/store/4718wmk03wr3554kmf09vy80vkdjvq56-python3.11-setuptools-69.1.1/lib/python3.11/site-packages/setuptools/command/build_py.py:207: _Warning: Package 'ics.parsers' is absent from the `packages` configuration.
!!

        ********************************************************************************
        ############################
        # Package would be ignored #
        ############################
        Python recognizes 'ics.parsers' as an importable package[^1],
        but it is absent from setuptools' `packages` configuration.

        This leads to an ambiguous overall configuration. If you want to distribute this
        package, please make sure that 'ics.parsers' is explicitly added
        to the `packages` configuration field.

        Alternatively, you can also rely on setuptools' discovery methods
        (for example by using `find_namespace_packages(...)`/`find_namespace:`
        instead of `find_packages(...)`/`find:`).

        You can read more about "package discovery" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

        If you don't want 'ics.parsers' to be distributed and are
        already explicitly excluding 'ics.parsers' via
        `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
        you can try to use `exclude_package_data`, or `include-package-data=False` in
        combination with a more fine grained `package-data` configuration.

        You can read more about "package data files" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/datafiles.html


        [^1]: For Python, any directory (with suitable naming) can be imported,
              even if it does not contain any `.py` files.
              On the other hand, currently there is no concept of package data
              directory, all directories are treated like packages.
        ********************************************************************************

!!
  check.warn(importable)
/nix/store/4718wmk03wr3554kmf09vy80vkdjvq56-python3.11-setuptools-69.1.1/lib/python3.11/site-packages/setuptools/command/build_py.py:207: _Warning: Package 'ics.serializers' is absent from the `packages` configuration.
!!

        ********************************************************************************
        ############################
        # Package would be ignored #
        ############################
        Python recognizes 'ics.serializers' as an importable package[^1],
        but it is absent from setuptools' `packages` configuration.

        This leads to an ambiguous overall configuration. If you want to distribute this
        package, please make sure that 'ics.serializers' is explicitly added
        to the `packages` configuration field.

        Alternatively, you can also rely on setuptools' discovery methods
        (for example by using `find_namespace_packages(...)`/`find_namespace:`
        instead of `find_packages(...)`/`find:`).

        You can read more about "package discovery" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

        If you don't want 'ics.serializers' to be distributed and are
        already explicitly excluding 'ics.serializers' via
        `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
        you can try to use `exclude_package_data`, or `include-package-data=False` in
        combination with a more fine grained `package-data` configuration.

        You can read more about "package data files" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/datafiles.html


        [^1]: For Python, any directory (with suitable naming) can be imported,
              even if it does not contain any `.py` files.
              On the other hand, currently there is no concept of package data
              directory, all directories are treated like packages.
        ********************************************************************************

!!
  check.warn(importable)
creating build/lib/ics/alarm
copying ics/alarm/__init__.py -> build/lib/ics/alarm
copying ics/alarm/audio.py -> build/lib/ics/alarm
copying ics/alarm/base.py -> build/lib/ics/alarm
copying ics/alarm/custom.py -> build/lib/ics/alarm
copying ics/alarm/display.py -> build/lib/ics/alarm
copying ics/alarm/email.py -> build/lib/ics/alarm
copying ics/alarm/none.py -> build/lib/ics/alarm
copying ics/alarm/utils.py -> build/lib/ics/alarm
creating build/lib/ics/grammar
copying ics/grammar/__init__.py -> build/lib/ics/grammar
copying ics/grammar/contentline.ebnf -> build/lib/ics/grammar
copying ics/grammar/contentline.py -> build/lib/ics/grammar
copying ics/grammar/parse.py -> build/lib/ics/grammar
creating build/lib/ics/parsers
copying ics/parsers/alarm_parser.py -> build/lib/ics/parsers
copying ics/parsers/attendee_parser.py -> build/lib/ics/parsers
copying ics/parsers/event_parser.py -> build/lib/ics/parsers
copying ics/parsers/icalendar_parser.py -> build/lib/ics/parsers
copying ics/parsers/parser.py -> build/lib/ics/parsers
copying ics/parsers/todo_parser.py -> build/lib/ics/parsers
creating build/lib/ics/serializers
copying ics/serializers/alarm_serializer.py -> build/lib/ics/serializers
copying ics/serializers/attendee_serializer.py -> build/lib/ics/serializers
copying ics/serializers/event_serializer.py -> build/lib/ics/serializers
copying ics/serializers/icalendar_serializer.py -> build/lib/ics/serializers
copying ics/serializers/serializer.py -> build/lib/ics/serializers
copying ics/serializers/todo_serializer.py -> build/lib/ics/serializers
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/__init__.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/__meta__.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/attendee.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/component.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/event.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/icalendar.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/timeline.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/todo.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/tools.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/types.py -> build/bdist.linux-x86_64/wheel/ics
copying build/lib/ics/utils.py -> build/bdist.linux-x86_64/wheel/ics
creating build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/__init__.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/audio.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/base.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/custom.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/display.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/email.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/none.py -> build/bdist.linux-x86_64/wheel/ics/alarm
copying build/lib/ics/alarm/utils.py -> build/bdist.linux-x86_64/wheel/ics/alarm
creating build/bdist.linux-x86_64/wheel/ics/grammar
copying build/lib/ics/grammar/__init__.py -> build/bdist.linux-x86_64/wheel/ics/grammar
copying build/lib/ics/grammar/contentline.ebnf -> build/bdist.linux-x86_64/wheel/ics/grammar
copying build/lib/ics/grammar/contentline.py -> build/bdist.linux-x86_64/wheel/ics/grammar
copying build/lib/ics/grammar/parse.py -> build/bdist.linux-x86_64/wheel/ics/grammar
creating build/bdist.linux-x86_64/wheel/ics/parsers
copying build/lib/ics/parsers/alarm_parser.py -> build/bdist.linux-x86_64/wheel/ics/parsers
copying build/lib/ics/parsers/attendee_parser.py -> build/bdist.linux-x86_64/wheel/ics/parsers
copying build/lib/ics/parsers/event_parser.py -> build/bdist.linux-x86_64/wheel/ics/parsers
copying build/lib/ics/parsers/icalendar_parser.py -> build/bdist.linux-x86_64/wheel/ics/parsers
copying build/lib/ics/parsers/parser.py -> build/bdist.linux-x86_64/wheel/ics/parsers
copying build/lib/ics/parsers/todo_parser.py -> build/bdist.linux-x86_64/wheel/ics/parsers
creating build/bdist.linux-x86_64/wheel/ics/serializers
copying build/lib/ics/serializers/alarm_serializer.py -> build/bdist.linux-x86_64/wheel/ics/serializers
copying build/lib/ics/serializers/attendee_serializer.py -> build/bdist.linux-x86_64/wheel/ics/serializers
copying build/lib/ics/serializers/event_serializer.py -> build/bdist.linux-x86_64/wheel/ics/serializers
copying build/lib/ics/serializers/icalendar_serializer.py -> build/bdist.linux-x86_64/wheel/ics/serializers
copying build/lib/ics/serializers/serializer.py -> build/bdist.linux-x86_64/wheel/ics/serializers
copying build/lib/ics/serializers/todo_serializer.py -> build/bdist.linux-x86_64/wheel/ics/serializers
running install_egg_info
Copying ics.egg-info to build/bdist.linux-x86_64/wheel/ics-0.7.2-py3.11.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/ics-0.7.2.dist-info/WHEEL
creating '/build/source/dist/.tmp-9sbpouf6/ics-0.7.2-py2.py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'ics/__init__.py'
adding 'ics/__meta__.py'
adding 'ics/attendee.py'
adding 'ics/component.py'
adding 'ics/event.py'
adding 'ics/icalendar.py'
adding 'ics/timeline.py'
adding 'ics/todo.py'
adding 'ics/tools.py'
adding 'ics/types.py'
adding 'ics/utils.py'
adding 'ics/alarm/__init__.py'
adding 'ics/alarm/audio.py'
adding 'ics/alarm/base.py'
adding 'ics/alarm/custom.py'
adding 'ics/alarm/display.py'
adding 'ics/alarm/email.py'
adding 'ics/alarm/none.py'
adding 'ics/alarm/utils.py'
adding 'ics/grammar/__init__.py'
adding 'ics/grammar/contentline.ebnf'
adding 'ics/grammar/contentline.py'
adding 'ics/grammar/parse.py'
adding 'ics/parsers/alarm_parser.py'
adding 'ics/parsers/attendee_parser.py'
adding 'ics/parsers/event_parser.py'
adding 'ics/parsers/icalendar_parser.py'
adding 'ics/parsers/parser.py'
adding 'ics/parsers/todo_parser.py'
adding 'ics/serializers/alarm_serializer.py'
adding 'ics/serializers/attendee_serializer.py'
adding 'ics/serializers/event_serializer.py'
adding 'ics/serializers/icalendar_serializer.py'
adding 'ics/serializers/serializer.py'
adding 'ics/serializers/todo_serializer.py'
adding 'ics-0.7.2.dist-info/LICENSE.rst'
adding 'ics-0.7.2.dist-info/METADATA'
adding 'ics-0.7.2.dist-info/WHEEL'
adding 'ics-0.7.2.dist-info/top_level.txt'
adding 'ics-0.7.2.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Successfully built ics-0.7.2-py2.py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for ics-0.7.2-py2.py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
@nix { "action": "setPhase", "phase": "installPhase" }
Running phase: installPhase
Executing pypaInstallPhase
Successfully installed ics-0.7.2-py2.py3-none-any.whl
Finished executing pypaInstallPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/75r8q414n0jvp3yy57bh2cw8c0322b0p-python3.11-ics-0.7.2
checking for references to /build/ in /nix/store/75r8q414n0jvp3yy57bh2cw8c0322b0p-python3.11-ics-0.7.2...
patching script interpreter paths in /nix/store/75r8q414n0jvp3yy57bh2cw8c0322b0p-python3.11-ics-0.7.2
stripping (with command strip and flags -S -p) in  /nix/store/75r8q414n0jvp3yy57bh2cw8c0322b0p-python3.11-ics-0.7.2/lib
shrinking RPATHs of ELF executables and libraries in /nix/store/nxjsh46v9cgh3xl9jqr5sihph141ib2m-python3.11-ics-0.7.2-dist
checking for references to /build/ in /nix/store/nxjsh46v9cgh3xl9jqr5sihph141ib2m-python3.11-ics-0.7.2-dist...
patching script interpreter paths in /nix/store/nxjsh46v9cgh3xl9jqr5sihph141ib2m-python3.11-ics-0.7.2-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: ics
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.11.8, pytest-8.0.2, pluggy-1.4.0
rootdir: /build/source
configfile: setup.cfg
testpaths: ics, tests
plugins: flakes-4.0.5
collecting ... 
collecting 172 items                                                           
collected 252 items / 5 deselected / 247 selected                              

ics/__init__.py .                                                        [  0%]
ics/__meta__.py .                                                        [  0%]
ics/alarm/__init__.py .                                                  [  1%]
ics/alarm/audio.py .                                                     [  1%]
ics/alarm/base.py .                                                      [  2%]
ics/alarm/custom.py .                                                    [  2%]
ics/alarm/display.py .                                                   [  2%]
ics/alarm/email.py .                                                     [  3%]
ics/alarm/none.py .                                                      [  3%]
ics/alarm/utils.py .                                                     [  4%]
ics/attendee.py .                                                        [  4%]
ics/component.py .                                                       [  4%]
ics/event.py .                                                           [  5%]
ics/grammar/__init__.py .                                                [  5%]
ics/grammar/contentline.py .                                             [  6%]
ics/grammar/parse.py .                                                   [  6%]
ics/icalendar.py .                                                       [  6%]
ics/parsers/alarm_parser.py .                                            [  7%]
ics/parsers/attendee_parser.py .                                         [  7%]
ics/parsers/event_parser.py .                                            [  8%]
ics/parsers/icalendar_parser.py .                                        [  8%]
ics/parsers/parser.py .                                                  [  8%]
ics/parsers/todo_parser.py .                                             [  9%]
ics/serializers/alarm_serializer.py .                                    [  9%]
ics/serializers/attendee_serializer.py .                                 [ 10%]
ics/serializers/event_serializer.py .                                    [ 10%]
ics/serializers/icalendar_serializer.py .                                [ 10%]
ics/serializers/serializer.py .                                          [ 11%]
ics/serializers/todo_serializer.py .                                     [ 11%]
ics/timeline.py .                                                        [ 12%]
ics/todo.py .                                                            [ 12%]
ics/tools.py .                                                           [ 12%]
ics/types.py .                                                           [ 13%]
ics/utils.py .                                                           [ 13%]
tests/__init__.py .                                                      [ 14%]
tests/alarm.py ........................                                  [ 23%]
tests/calendar.py .....................                                  [ 32%]
tests/component.py .................                                     [ 39%]
tests/contentline.py ....F                                               [ 41%]
tests/duration.py ....                                                   [ 42%]
tests/event.py ......................................................... [ 65%]
.....                                                                    [ 68%]
tests/fixture.py .                                                       [ 68%]
tests/misc.py .............                                              [ 73%]
tests/parse.py .......                                                   [ 76%]
tests/test.py ..                                                         [ 77%]
tests/timeline.py .........                                              [ 80%]
tests/todo.py ........................                                   [ 90%]
tests/unfold_lines.py ............                                       [ 95%]
tests/utils.py ...........                                               [100%]
```
</details>

```console
=================================== FAILURES ===================================
__________________ TestContentLine.test_timezone_not_dropped ___________________

self = <tests.contentline.TestContentLine testMethod=test_timezone_not_dropped>

    def test_timezone_not_dropped(self):
        line = ContentLine.parse("DTSTART;TZID=Europe/Berlin:20151104T190000")
        arrow = iso_to_arrow(line)
>       self.assertIn("Europe/Berlin", str(arrow.tzinfo))
E       AssertionError: 'Europe/Berlin' not found in "tzfile('Atlantic/Jan_Mayen')"

tests/contentline.py:83: AssertionError
=============================== warnings summary ===============================
../../nix/store/m34m9sb8z84ldimp3wzwwrz08l7w66ly-python3.11-pytest-8.0.2/lib/python3.11/site-packages/_pytest/config/__init__.py:1396
  /nix/store/m34m9sb8z84ldimp3wzwwrz08l7w66ly-python3.11-pytest-8.0.2/lib/python3.11/site-packages/_pytest/config/__init__.py:1396: PytestConfigWarning: Unknown config option: pep8ignore
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

tests/alarm.py: 6 warnings
tests/calendar.py: 3 warnings
tests/component.py: 2 warnings
tests/event.py: 14 warnings
tests/todo.py: 3 warnings
  /build/source/ics/component.py:85: FutureWarning: Behaviour of str(Component) will change in version 0.9 to only return a short description, NOT the ics representation. Use the explicit Component.serialize() to get the ics representation.
    warnings.warn(

tests/calendar.py::TestCalendar::test_iter
tests/calendar.py::TestCalendar::test_iter
tests/calendar.py::TestCalendar::test_iter
  /build/source/ics/icalendar.py:104: DeprecationWarning: Using Calendar as Iterable is deprecated and will be removed in version 0.8. Use the explicit calendar.serialize_iter() instead.
    warnings.warn(

tests/misc.py::TestEvent::test_fixtures_time
  /build/source/ics/component.py:47: UserWarning: The VERSION property was not found and is required by the RFC. A default value of "VERSION:2.0" has been used instead
    warnings.warn(message)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/contentline.py::TestContentLine::test_timezone_not_dropped - AssertionError: 'Europe/Berlin' not found in "tzfile('Atlantic/Jan_Mayen')"
=========== 1 failed, 246 passed, 5 deselected, 33 warnings in 1.85s ===========
/nix/store/v5lsd029lz5lfhamivbgqyp3zdv94ah2-stdenv-linux/setup: line 1578: pop_var_context: head of shell_variables not a function context
```

### Additional context

This is for version 0.7.2 — 0.8.0 is not yet available, and the failing test appears to have been removed.
I think this is a NixOS-specific issue, but I've raised ics-py/ics-py#414 as there may be implications for upstream.

### Notify maintainers

@jonringer (there's been no maintainer for two years and you merged the last update.)

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix run 'nixpkgs#nix-info' -- -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.22, NixOS, 24.05 (Uakari), 24.05.20240319.b06025f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - channels(db): `""`
 - nixpkgs: `/nix/store/zaza7mgggz4m5h6z18kajabf4wciaj47-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by jonringer** [comment]
Sorry, [I was banned for 6 weeks over having the wrong viewpoints](https://discourse.nixos.org/t/why-was-jon-ringer-banned-from-github/44114/23).

Looks like this was fixed in 17f73d6f9ade07a11baf897b902e99b43d41ebea

**Comment by dbaynard** [comment]
I'm often a little unsure whom to tag in these issues. Seven weeks is very much on the shorter end for a response to such an issue, so thanks! And welcome back.
